ql/src/test/results/clientpositive/llap/mapjoin_hint.q.out

```diff
@@ -648,7 +651,7 @@ STAGE PLANS:
                         outputColumnNames: _col0, _col1, _col2, _col3, _col4, _col5, _col6, _col7, _col8
                         input vertices:
                           1 Reducer 3
-                        Statistics: Num rows: 1 Data size: 816 Basic stats: COMPLETE Column stats: NONE
+                        Statistics: Num rows: 0 Data size: 0 Basic stats: NONE Column stats: NONE
```

Comment:
it's great that now we are able to deduce that there will be 0 rows...
I'm wondering if there will be any sideeffects of this `Basic stats: NONE`.

Reply:
@kgyrtkirk I guess Basic stats:NONE is not really correct since we do have full stats here. But since number of rows is zero I don't think there should be any side effects.